Automattic/jetpack
# Social: Add UTM settings and toggles

Fixes https://github.com/Automattic/jetpack-reach/issues/674

## Proposed changes:

* Adds a new setting and toggle for it to the Jetpack Social admin page and to Jetpack settings
* This setting will be used to toggle the UTM params on the sites

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to JP settings and to the Social Admin page
* Test that the toggle works as expected
* Also check the option on the local backend

![CleanShot 2024-11-04 at 14 36 38 png](https://github.com/user-attachments/assets/713b3165-4039-494a-ad3a-dd51a6b1dd99)

![CleanShot 2024-11-04 at 14 36 20 png](https://github.com/user-attachments/assets/26963d49-c565-4641-8033-f25b9df73c63)
